### DISCUSSION TITLE
Possible results for "kind"

### DISCUSSION THREAD

**Discussion by nineclicks** [cinemagoer/cinemagoer#368] [Q&A] [answered]
Is there a list of all of the possible results for a title's "kind" e.g. 'movie', 'tv movie', 'video movie'. I am not turning up anything searching the docs.

**Comment by alberanid** [comment, marked as answer]
not exactly.  You can get the list from the values of https://github.com/alberanid/imdbpy/blob/a89e9f499d7248046be05a8a21ac26c9b6bb70a7/imdb/utils.py#L57 (imdb.utils.KIND_MAP) plus the default "movie".

Some types are still missing; see "title type" at https://www.imdb.com/search/title/
